docs/README.md

```diff
@@ -0,0 +1,37 @@
+# OneFuzz Documentation
```

Comment:
any suggestions on how to improve this?

Reply:
I think it's a great start! It made me realize we're lacking in our user facing 'how-to' style guides.